delete_main.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+"""
+    token_get.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Token` module: Fetch token of type `csrf`
+    MIT license
+"""
+
+import requests
+
+S = requests.Session()
+
+URL = "https://mediawiki.org/w/api.php"
+
+PARAMS_0 = {
+    'action':"query",
+    'meta':"tokens",
+    'type':"csrf",
+    'format':"json"
+}
+
+R = S.get(url=URL, params=PARAMS_0)
+DATA = R.json()
+
+CSRF_TOKEN = DATA['query']['tokens']['csrftoken']
+
+print(CSRF_TOKEN)
+
+
+# after getting the csrf toke above, we then
```

Comment:
You could rephrase this as "Use the csrf token in a post request to delete a page"